code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -234,7 +234,7 @@
 			if(M_ELVIS in mutations)
 				src.emote("fart")
 				msg = "<B>[src]</B> has left the building..."
-			if(M_HARDCORE in mutations)
+			if(!miming && M_HARDCORE in mutations)
```

Comment:
I still cannot fathom why it was decided that `in` should have a lower priority than *any* other operator
Let alone *all* of them